test/integration/spec/remotetracks.js

```diff
@@ -34,6 +35,234 @@ function getTracksOfKind(participant, kind) {
 }
 
+(defaults.largeRoom ? describe : describe.skip)('RemoteAudioTrack', function() {
+  // eslint-disable-next-line no-invalid-this
+  this.timeout(60000);
+
+  describe('when subscribed to by the LocalParticipant', () => {
+    let aliceRemoteAudioTracks;
+    let aliceRemoteForBob;
+    let aliceRemoteForCharlie;
+    let aliceRoom;
+    let bobRoom;
+    let charlieRoom;
+
+    before(async () => {
+      [, aliceRoom, [bobRoom, charlieRoom]] = await waitFor(setup({
+        testOptions: { audio: true, video: false },
+        otherOptions: { audio: false, video: false },
+        participantNames: ['alice', 'bob', 'charlie'],
+        nTracks: 0
+      }), 'Alice, Bob and Charlie to join a Room');
+
+      const { localParticipant: { sid: aliceParticipantSid } } = aliceRoom;
+      aliceRemoteForBob = bobRoom.participants.get(aliceParticipantSid);
+      aliceRemoteForCharlie = charlieRoom.participants.get(aliceParticipantSid);
+      return waitFor([
+        tracksSubscribed(aliceRemoteForBob, 1),
+        tracksSubscribed(aliceRemoteForCharlie, 1)
+      ], 'Bob and Charlie to subscribe to Alice\'s AudioTrack');
+    });
+
+    it('should initially be switched off with the reason "disabled-by-subscriber"', () => {
+      aliceRemoteAudioTracks = [aliceRemoteForBob, aliceRemoteForCharlie].map(aliceRemote =>
+        aliceRemote.audioTracks.values().next().value.track);
+      assert.equal(aliceRemoteAudioTracks.length, 2);
+      aliceRemoteAudioTracks.forEach(aliceRemoteAudioTrack => {
+        assert.equal(aliceRemoteAudioTrack.isSwitchedOff, true);
+        assert.equal(aliceRemoteAudioTrack.mediaStreamTrack, null);
+        assert.equal(aliceRemoteAudioTrack.switchOffReason, 'disabled-by-subscriber');
+      });
+    });
+
+    it('should later be switched on', async () => {
+      await waitFor(
+        aliceRemoteAudioTracks.map(trackSwitchedOn),
+        'Alice\'s RemoteAudioTracks to be switched on'
+      );
+      aliceRemoteAudioTracks.forEach(aliceRemoteAudioTrack => {
+        assert.equal(aliceRemoteAudioTrack.isSwitchedOff, false);
+        assert(aliceRemoteAudioTrack.mediaStreamTrack instanceof MediaStreamTrack);
+        assert.equal(aliceRemoteAudioTrack.switchOffReason, null);
+      });
+    });
+
+    after(() => {
+      [aliceRoom, bobRoom, charlieRoom].forEach(room => room && room.disconnect());
+    });
+  });
+
+  // TODO(mmalavalli) Enable test once SFU implements this behavior.
+  describe.skip('enable/disable', () => {
+    let aliceRoom;
+    let bobRoom;
+    let charlieRoom;
+    let daveRoom;
+
+    before(async () => {
+      [, aliceRoom, [bobRoom, charlieRoom]] = await waitFor(setup({
+        testOptions: { audio: true, video: false },
+        otherOptions: { audio: true, video: false },
+        participantNames: ['Alice', 'Bob', 'Charlie'],
+        nTracks: 1
+      }), 'Alice, Bob and Charlie to join a Room');
```

Comment:
message should include come context data - like room sid, participant sid - without this it becomes hard to debug after seeing messages on circleci

Reply:
@makarandp0 At this stage we don't have access to any of that information.